auctionsystem/tcp/client.py

```diff
@@ -62,13 +64,17 @@ def get_tcp_client_socket():
             sys.exit()
 
     async def _handle_receive(self, loop, handle_receive_cb):
-        while True:
-            data = await loop.sock_recv(self._socket, 1024)
-            print('Received: {0} from {1}'.format(data, self.server_address), file=sys.stderr)
-            handle_receive_cb(data, self.server_address)
+        while not self._closed:
+            try:
+                data = await loop.sock_recv(self._socket, 1024)
+                print('Received: {0} from {1}'.format(data, self.server_address), file=sys.stderr)
+                handle_receive_cb(data, self.server_address)
+            except OSError:
+                await asyncio.sleep(0.1)
+                continue
```

Comment:
If there isn't anything after this `except` clause, couldn't `continue` be removed?